gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/MRCompactor.java

```diff
@@ -364,7 +366,13 @@ private GobblinMetrics initializeMetrics() {
     tags.addAll(Tag.fromMap(ClusterNameTags.getClusterNameTags()));
     GobblinMetrics gobblinMetrics =
         GobblinMetrics.get(this.state.getProp(ConfigurationKeys.JOB_NAME_KEY), null, tags.build());
-    gobblinMetrics.startMetricReporting(this.state.getProperties());
+    try {
```

Comment:
Just FYI, this file is deprecated and not being used anywhere internally AFAIK.

Reply:
Thanks! Yes, indeed this class is deprecated. The exception handling has been added here to preserve completeness, more than anything else.